test/app-tap/uri.test.lua

```diff
@@ -160,6 +230,20 @@ local function test_format(test)
             "unix/:/tmp/unix.sock", "URI format")
 end
 
+local function test_format_ipv6_format_by_rfc2732(test)
+    test:plan(2)
+    local u
+
+    -- URI with IPv6
```

Comment:
missed dot at the end of comment. here and below

Reply:
This comment has been removed